browser/layout/pa11y-config.js

```diff
@@ -6,19 +6,19 @@ module.exports = {
 	pa11yData: [
 		{
 			rootElement: 'html',
-			hideElements: '.n-skip-link',
+			hideElements: '.n-skip-link, iframe[src*=doubleclick]',
```

Comment:
Why are we targeting the `iframe` and also using the `enableGTM` flag?

Reply:
Sorted now